components/motor/gpio/motor_encoder.go

```diff
@@ -59,6 +59,8 @@ func WrapMotorWithEncoder(
 	if e == nil {
 		return m, nil
 	}
+
+	logger.Info("meow")
```

Comment:
assuming this is for debugging, but remove before merging

Reply:
yes. Was verifying that the server I sent over mutagen was the server I built, since none of the other changes show up when a new binary is built.

Reply:
meow